geppetto.js/geppetto-ui/src/camera-controls/CameraControls.js

```diff
@@ -143,14 +140,18 @@ class CameraControls extends Component {
         icon: null,
       });
     }
+
+    const buttonStyles = { color: '#fc6320', }

```

Comment:
this does not work right now, it is hardcoded for geppetto but applications are customising the color of these controls

Reply:
Correct, this should be customizable through props. Thanks for the review :+1: